### ISSUE TITLE
Missing ALT Text  - Information Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OSHA-Heat-Safety-iOS#4]
Missing image ALT text for all the images in the information section.

WCAG 2.0 - WCAG 2.0 - 1.1.1 Non-text Content - Level A

**Comment by skeeJay** [comment]
There is no alt text for this content in the current version of the app, so this isn't a regression. I'd suggest addressing in a future version, as this would likely require new localized copy.

**Comment by jfc3-dol** [comment]
I can create the ALT text or accessibility labels for the images, whatever the term is for iOS applications.

**Comment by skeeJay** [comment]
(In the case of the precaution and more info content views, they're implemented as local HTML webviews, so they are indeed ALT text.)

I can easily add in ALT text for these images, if we have it. But we'll needs Spanish versions as well, for the Spanish version of each of the 9 content pages.

**Comment by jfc3-dol** [comment]
Thanks, for adding the English and Spanish ALT text to the images as requested.

Closing this issue.
